### PR TITLE
[WIP] patch error handling at vm

### PR DESCRIPTION
**Pull request by ramtinms**
I'm not 100% sure if this fixes the issue and it might cause forking so we should review.

### CODE REVIEW

File: fvm/env.go [hunk 1]
```diff
@@ -297,7 +297,8 @@ func (e *hostEnv) VerifySignature(
 
 	if err != nil {
 		// TODO: improve error passing https://github.com/onflow/cadence/issues/202
-		panic(err)
+		// panic(err)
```
Comment (turbolent) [inline fvm/env.go:300]: Maybe replace this by a TODO

**Review by turbolent — APPROVED** [review]

**Comment by jordanschalm** [comment]
Does this panic get caught? I don't think the exe node is crashing on devnet, it's logging an error and failing to execute a block

**Review by jordanschalm — APPROVED** [review]
I'm good to merge this, but I don't think it is related to the testnet issues we saw today.

**Comment by ramtinms** [comment]
Yeah, I'm going to close this, anyway it is going to be addressed in the future FVM PRs. no need to patch it here.

**Pull request closed without merging**